tests/src/system/rest/GoCLINginxTests.scala

```diff
@@ -81,6 +82,13 @@ class GoCLINginxTests extends FlatSpec with Matchers with RestUtil {
         }
     }
 
+    it should "respond to {$DownloadLinkGoCli}/content.json" in {
+        val response = RestAssured.given().config(sslconfig).
+          get(getServiceURL() + DownloadLinkGoCli + "/content.json")
+
+        response.statusCode should be(200)
```

Comment:
Now with content.json, maybe the existing tests should be driven by the content of content.json
For example check if you can download mac and the location of the binary you get from content.json